### PR TITLE
Formatting and small grammar and latex fixes.

### PR DESCRIPTION
**Pull request by gdevenyi**

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -179,8 +179,7 @@ \section*{Introduction}
 ``intermediate'',
 and ``advanced''
 to help authors write at appropriate levels.
-However,
-this only works if there is prior agreement on what those terms mean.
+This method works, however, only if there is prior agreement on what those terms mean.
```
Comment (gvwilson) [inline paper.tex]: Why?

File: paper.tex [hunk 2]
```diff
@@ -191,7 +190,7 @@ \section*{Introduction}
 Learner profiles have five parts:
 the learner's general background,
 what they already know,
-what *they* think they want to do,
+what \emph{they} think they want to do,
```
Comment (gvwilson) [inline paper.tex]: good catch

File: paper.tex [hunk 3]
```diff
@@ -269,8 +268,7 @@ \section*{Introduction}
 Before re-using content,
 make sure to check that the lesson is licensed in an open manner.
 Both Programming Historian and Software Carpentry license their material under
-the Creative Commons - Attribution license
-(see https://creativecommons.org/licenses/by/2.0/).
+the Creative Commons - Attribution license\footnote{\url{https://creativecommons.org/licenses/by/2.0/}}.
```
Comment (gvwilson) [inline paper.tex]: we can't footnote so `(\url{...})` here please

File: paper.tex [hunk 4]
```diff
@@ -359,7 +357,7 @@ \section*{Introduction}
 If people have a shared understanding of how lessons should be developed,
 it is easier for them to work together.
 Less obviously,
-if people have a shared model of how lessons are going to be *used*,
+if people have a shared model of how lessons are going to be \emph{used},
```
Comment (gvwilson) [inline paper.tex]: good catch

File: paper.tex [hunk 5]
```diff
@@ -449,7 +447,7 @@ \section*{Introduction}
 and it is worth spending some time on automating this release process.
 After the formal publishing,
 it is also a good idea to publish the built lesson for direct access by the community,
-making it clear which version is displayed and what is its corresponding DOI. 
+making it clear which version is displayed and what is its corresponding DOI@.
```
Comment (gvwilson) [inline paper.tex]: What's the `@` for?

File: paper.tex [hunk 6]
```diff
@@ -510,7 +508,7 @@ \section*{Introduction}
 If there are several complementary ways to explain something,
 or several points of views that can cohabit respectfully,
 it may be possible to present them side by side.
-There are actually good pedagogical reasons to do this even if contributors *don't* disagree:
+There are actually good pedagogical reasons to do this even if contributors \emph{don't} disagree:
```
Comment (gvwilson) [inline paper.tex]: Good catch.

File: paper.tex (hunk 1 above)
Comment (gdevenyi) [inline paper.tex]: http://www.bartleby.com/141/strunk3.html, Ctrl-F "however"

At least as far as I read it.

File: paper.tex (hunk 5 above)
Comment (gdevenyi) [inline paper.tex]: Tells LaTeX that "DOI." is the end of a sentence, not an abbreviation.

**Comment by gdevenyi** [comment]
So, since this didn't get merged, looks like people did it already.

Oh well.

File: paper.tex (hunk 1 above)
Comment (gvwilson) [inline paper.tex]: Hm. Not what I was taught, but who am I to disagree with S&W?

File: paper.tex (hunk 5 above)
Comment (gvwilson) [inline paper.tex]: Huh - been using LaTeX for 30 years, and I've never seen that.

**Pull request merged**